intelmq/tests/bots/parsers/shadowserver/test_sandbox_dns.py

```diff
@@ -0,0 +1,92 @@
+# SPDX-FileCopyrightText: 2019 Guillermo Rodriguez
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# -*- coding: utf-8 -*-
+
+import os
+import unittest
+
+import intelmq.lib.test as test
+import intelmq.lib.utils as utils
+from intelmq.bots.parsers.shadowserver.parser import ShadowserverParserBot
+
+with open(os.path.join(os.path.dirname(__file__),
+                       'testdata/sandbox_dns.csv')) as handle:
+    EXAMPLE_FILE = handle.read()
+EXAMPLE_LINES = EXAMPLE_FILE.splitlines()
+
+EXAMPLE_REPORT = {'feed.name': 'Sandbox DNS',
+                  "raw": utils.base64_encode(EXAMPLE_FILE),
+                  "__type": "Report",
+                  "time.observation": "2022-01-07T00:00:00+00:00",
+                  "extra.file_name": "2022-01-07-sandbox_dns-test.csv",
+                  }
+EVENTS = [
+        {
+   '__type' : 'Event',
+   'classification.identifier' : 'sandbox-dns',
+   'classification.taxonomy' : 'other',
+   'classification.type' : 'other',
+   'extra.request' : 'time.windows.com',
+   'extra.response' : '40.119.6.228',
+   'extra.type' : 'A',
```

Comment:
```suggestion
   'extra.dns_query_type' : 'A',
```

Reply:
Updated in 655055d169257c1f93ee18bda28bdc28600ed5ed